lib/core.js

```diff
@@ -555,6 +555,8 @@ Archiver.prototype.append = function(source, data) {
  * @return {this}
  */
 Archiver.prototype.bulk = function(mappings) {
+  console.log('Archiver.bulk() deprecated since 0.21.0');
```

Comment:
The `console.warn` function is an alias for `console.error` according to docs, so no difference in those. I didn't know about the `process.emitWarning` so I will probably switch it over to that and figure out how to gracefully fall back to `console.warn` when run on older versions. There also appears to be a `DeprecationWarning` for the emitting which can be useful.